tock/hours/models.py

```diff
@@ -314,6 +319,17 @@ def calculate_target_hours(self):
         this_weeks_target = round(self.billable_expectation * self.calculate_utilization_denominator(), 0)
         return min(this_weeks_target, self.max_target_hours())
 
+    def calculate_total_weekly_allocation(self):
+        """
+        Loops through related time card objects and sums total
+        weekly allocation percentage
+        """
+        timecardobjs = self.timecardobjects.all()
+        total_weekly_allocation = 0.00
+        for tco in timecardobjs:
+            total_weekly_allocation += float(tco.project_allocation)
+        return total_weekly_allocation
```

Comment:
Do we need a test here? Are there any edge cases with DecimalField that we need to account for when summing up? I don't think so but wanted to make sure I wasn't missing something here.

Reply:
I don't believe so, because the default for `project_allocation` is `0.00`

Reply:
Right you are! Sounds good then, no concerns now.